developer/src/kmc-generate/src/template/kmn-keyboard/source/keyboard.kvks

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<visualkeyboard>
+  <header>
+    <version>10.0</version>
+    <kbdname>$ID</kbdname>
+    <flags/>
+  </header>
+</visualkeyboard>
```

Comment:
I think I saw the other template files were based on US layout. Does kvks need default keycaps?

Reply:
The .kvks keycaps do not need to be included here. It's up to the keyboard developer whether they add defaults or base keycaps or not; these tweaks can be made in the visual keyboard editor.